Index: .gradle/config.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#Wed Aug 14 14:55:58 MSK 2024\njava.home=D\\:\\\\android\\\\jbr\n
===================================================================
diff --git a/.gradle/config.properties b/.gradle/config.properties
--- a/.gradle/config.properties	
+++ b/.gradle/config.properties	
@@ -1,2 +1,2 @@
-#Wed Aug 14 14:55:58 MSK 2024
+#Wed Aug 14 17:01:02 MSK 2024
 java.home=D\:\\android\\jbr
